CONTRIBUTORS

```diff
@@ -12,7 +12,6 @@ CRoed <chris.roed@anserinnovation.com>
 Dave Gramlich <callmehiphop@gmail.com>
 Eric Uldall <ericuldall@gmail.com>
 F. Hinkelmann <fhinkel@vt.edu>
-Franziska Hinkelmann <franziska.hinkelmann@gmail.com>
```

Comment:
You can make this change permanant by adding a `.mailmap` file with the following contents:

```
F. Hinkelmann <fhinkel@vt.edu> Franziska Hinkelmann <franziska.hinkelmann@gmail.com>
```

Reply:
Thanks.